tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3618,8 +3596,6 @@ public function groupExists(string $group):bool {
 		}
 		if (OcisHelper::isTestingWithGraphApi()) {
 			$base = '/graph/v1.0';
-		} else {
-			$base = '/ocs/v2.php/cloud';
 		}
 		$group = \rawurlencode($group);
 		$fullUrl = $this->getBaseUrl() . "$base/groups/$group";
```

Comment:
```suggestion
		$fullUrl = $this->getBaseUrl() . "/graph/v1.0/groups/$group";
```
@saw-jan can we do this